torch/package/package_exporter.py

```diff
@@ -197,7 +197,7 @@ def __init__(
             self.buffer = f
 
         self.zip_file = torch._C.PyTorchFileWriter(f)
-        self.zip_file.set_min_version(6)
+        self.zip_file.set_version(6)
```

Comment:
tagging @suo for this change. Since we don't rely on dynamic versioning anymore, we no longer need to take the min of different modules.